collaboration/1.slack.md

```diff
@@ -0,0 +1,102 @@
+# Slack
+
+[Slack](https://slack.com/) is a chat we use to discuss about anything.
+
+You can join a conversation in a public channel or talk privately to a co-worker and it's a great way to keep the conversation while maintaining the office a quiet place.
+
+Slack is a powerful tool and we've been using it for many years now. Here are some of the things we learned from experience.
+
+## Private vs Public
+
+If you're in a rush, here's the main gist: **always use public channels**, unless you have a very good reason.
+
+If you want to know something more, read on.
+
+### Public channels
+Team awareness is extremely important, so - by default - prefer writing in a public channel over a private communication.
+Even if you are talking to a specific person, many times the information will be useful to other people, even when you don't realize it.
+You can still direct your question to a specific person using [mentions](https://get.slack.help/hc/en-us/articles/205240127-Mention-a-team-member).
+
+Remember that you can create new public channels easily, so if you are afraid to "spam" people you can create a new channel for the topic you want to discuss and move the conversation there. These channels can then be archived when their purpose is over (e.g. a channel about an event, that is now passed).
+Interested people can read the discussion in real-time or at a later time, and the information doesn't get lost in private conversations.
+
+Also, if you think creating a new channel is "too much", you can start a thread from a specific message, so that you don't clog the main stream of the channel.
```

Comment:
bold the **thread** word to understand the topic for each paragraph? Maybe I'm exaggerating :p but I think it's useful